Formula/metabase.rb

```diff
@@ -21,6 +21,8 @@ class Metabase < Formula
   def install
     if build.head?
       Language::Node.setup_npm_environment
+      # help yarn find npm's node-gyp binary inside superenv
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/lib/node_modules/npm/bin/node-gyp-bin"
```

Comment:
>  which is imho harmful (nothing should manipulate another formulas cellar).

Agreed. On a practical level, also impossible these days with the sandbox in place. Thanks for creating the issue around `yarn`, I'll go read how that discussion went & chuck in any thoughts I have there.